axon-server-connector/src/test/java/org/axonframework/axonserver/connector/util/ErrorCodeDeciderTest.java

```diff
@@ -0,0 +1,38 @@
+package org.axonframework.axonserver.connector.util;
+
+import org.axonframework.axonserver.connector.ErrorCode;
+import org.axonframework.serialization.SerializationException;
+import org.junit.jupiter.api.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Unit tests for {@link ErrorCodeDecider}
+ * @author Milan Savic
+ */
+class ErrorCodeDeciderTest {
```

Comment:
If you feel these operations belong in the `ErrorCode` enumeration, I think these tests can be added to the `ErrorCodeTest` class too.